tests/scripts/thread-cert/message.py

```diff
@@ -294,8 +306,30 @@ def next_coap_message(self, code, uri_path=None, assert_enabled=True):
 
         return message
 
+    #Get the newest mle message
+    def last_mle_message(self, command_type, assert_enabled = True):
+        message = None
+        size = len(self.messages)
+
+        for i in range(size - 1, -1, -1):
+            m = self.messages[i]
+
+            if m.type != MessageType.MLE:
+                continue
+
+            #for command_type in command_types:
+            if m.mle.command.type == command_type:
+                message = m
+                break
+
+        if assert_enabled:
+            assert message is not None, "Could not find CoapMessage with code: {}".format(command_type)
```

Comment:
here `CoapMessage with code` should be `MleMessage with type`?

Reply:
fixed